joomla-projects/gsoc22_accessibility-plugin
# Made feedback & accessibility redirection from backend

### Summary of Changes
Added the functionality for feedback redirection link & Accessibility statement. Site admins can select the Menu item where they want to redirect the user to report accessibility-related problems, and also for accessibility statements.

### Testing Instructions

1. Apply the patch.
2. Change the accessibility to https://github.com/joomla-projects/gsoc22_accessibility-plugin-script/tree/feedback in the package.json file.
3. Run npm install
4. Check that you can select the Menu Items for Accessibility statement and Accessibility Feeedback in the Accessibility plugin backend and the selected menu items are available for redirection in the the a11y plugin. 

Actual result BEFORE applying this Pull Request

![image](https://user-images.githubusercontent.com/54266411/187882289-79a0ec88-10e1-49b1-8212-beac945051a0.png)

![image](https://user-images.githubusercontent.com/54266411/187882413-8b58d887-8dce-4dcb-be27-946831d0c1ee.png)


Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/54266411/187882484-b33b07b7-ca85-4281-aea8-1e4b32aa92ed.png)

![image](https://user-images.githubusercontent.com/54266411/187882528-e5b49a12-1eff-4d2d-930c-8fa54570ede6.png)
